dask.yaml

```diff
@@ -4,12 +4,9 @@ optimization:
 
 distributed:
   version: 2
-  worker:
-    profile:
-      interval: 10s        # Time between statistical profiling queries
-      cycle: 1000s         # Time between starting new profile
-      low-level: false     # Whether or not to include low-level functions
   admin:
     event-loop: uvloop
-    system-monitor:
-      interval: 1h
+
+benchmark:
+  software: "gjoseph92/scheduler-benchmark"
+  checks: true
```

Comment:
What does this do Gabe?

Reply:
It toggles these: https://github.com/quasiben/dask-scheduler-performance/blob/769a0fa2332194388307b157b2f132c12efaa055/nightly-benchmark/nightly-run.py#L118-L120

Could give it more descriptive name if we want.